recipes/openbabel/2.4.1/meta.yaml

```diff
@@ -35,12 +37,15 @@ requirements:
         - zlib
         - libxml2
         - cairo   # [linux]
+        - xorg-libxau
```

Comment:
Our upstream channel does not require `xorg-*`, see https://travis-ci.org/openbabel/conda-openbabel/jobs/216283344#L368 

If we explicitly disable GUI it will not link libraries against it. Apparently bioconda's docker image has wxWidgets and the GUI is compiled by default.